tox.ini

```diff
@@ -9,39 +9,51 @@ setenv =
 passenv=
     PYTEST_ADDOPTS
 
-[flake_env]
-envdir = {toxworkdir}/.flake_env
-deps =
-    flake8==3.9.2
-    pylint==2.11.1
-    black==21.12b0
-    isort==5.10.1
-    -r{toxinidir}/test-requirements.txt
-    -r{toxinidir}/integration-requirements.txt
+[version]
+flake8==3.9.2
+pylint==2.11.1
+black==21.12b0
+isort==5.10.1
 
 [testenv:flake8]
-envdir = {[flake_env]envdir}
-deps = {[flake_env]deps}
+deps =
+    flake8=={[version]flake8}
 commands = {envpython} -m flake8 {posargs:cloudinit/ tests/ tools/ setup.py}
 
 [testenv:pylint]
-envdir = {[flake_env]envdir}
-deps = {[flake_env]deps}
+deps =
+    pylint=={[version]pylint}
+    -r{toxinidir}/test-requirements.txt
+    -r{toxinidir}/integration-requirements.txt
 commands = {envpython} -m pylint {posargs:cloudinit tests tools}
 
 [testenv:black]
-envdir = {[flake_env]envdir}
-deps = {[flake_env]deps}
+deps =
+    black=={[version]black}
 commands = {envpython} -m black . --check
 
 [testenv:isort]
-envdir = {[flake_env]envdir}
-deps = {[flake_env]deps}
+deps =
+    isort=={[version]isort}
 commands = {envpython} -m isort . --check-only
 
-[testenv:format]
-envdir = {[flake_env]envdir}
-deps = {[flake_env]deps}
+[testenv:check_format]
+envlist = flake8, pylint, black, isort
+deps =
+    flake8=={[version]flake8}
+    pylint=={[version]pylint}
+    black=={[version]black}
+    isort=={[version]isort}
+    -r{toxinidir}/test-requirements.txt
+    -r{toxinidir}/integration-requirements.txt
+commands =
```

Comment:
looks like copypasta of the `do_format` command. I think instead we want something like
```
commands =
    {[testenv:flake8]commands}
    {[testenv:pylint]commands}
    {[testenv:black]commands}
    {[testenv:isort]commands}
```